Automattic/woocommerce-payments
# Fix duplicated payment method on thank you page when using WooPay

Fixes 1890-gh-Automattic/woopay

#### Changes proposed in this Pull Request

When paying with WooPay, in the order received (thank you) page:

- Removes the success notice with the message `Thank you! We've received your order`. The message is still there but it's from the default template from WooCommerce.
- Removes the duplicated `Payment method` section (view image).

![244772643-7025f479-46c8-4028-a01d-b6b5da56358d](https://github.com/Automattic/woocommerce-payments/assets/4153905/1a9dac65-5fe5-4044-b8f5-947acb9b9c19)

#### Testing instructions

- Perform a transaction in the `develop` branch and pay with WooPay. Identify the duplicated `Payment method` section.
- Repeat the operation on this branch and identify that the duplicated section is gone, and the success notice (green background) was removed. _The text is still there, it's just not styled as a success notice._

#### Demo

<img width="394" alt="Screen Shot 2023-07-13 at 17 22 55 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4153905/a8e79a34-36e2-46a4-b167-8370b967b4cf">
<img width="1087" alt="Screen Shot 2023-07-13 at 17 22 18 PM" src="https://github.com/Automattic/woocommerce-payments/assets/4153905/4c14eeb4-88ed-4b20-b4b9-77497441a1cf">



-------------------


**Post merge**
